container-disc/src/main/java/com/yahoo/container/jdisc/ConfiguredApplication.java

```diff
@@ -116,6 +127,20 @@ public ConfiguredApplication(ContainerActivator activator,
         Container.get().setOsgi(new OsgiImpl(osgiFramework));
 
         applicationWithLegacySetup = new ContainerDiscApplication(configId);
+
+        slobrokRegistrator = registerInSlobrok(slobroksConfig, qrConfig);
+    }
+
+    /** The container has no rpc methods, but we still need to register it in Slobrok to enable orchestration */
+    private Register registerInSlobrok(SlobroksConfig slobrokConfig, QrConfig qrConfig) {
+        SlobrokList slobrokList = new SlobrokList();
+        slobrokList.setup(slobrokConfig.slobrok().stream().map(SlobroksConfig.Slobrok::connectionspec).toArray(String[]::new));
+        Spec mySpec = new Spec(HostName.getLocalhost(), qrConfig.rpc().port());
+        Register slobrokRegistrator = new Register(new Supervisor(new Transport()), slobrokList, mySpec);
+        slobrokRegistrator.registerName(qrConfig.rpc().slobrokId());
+        log.log(LogLevel.INFO,
```

Comment:
I assume this method is a NOP when there are no slobroks, except for the log message?

Reply:
I assume so too, but why do you ask? I think we only have two cases: Standalone, where the model will not enable rpc (as far as I can tell) and so not make it to this code, and clustered, where there are always slobroks.

Reply:
I forgot for a moment that there is a separate StandaloneContainerApplication.